### DISCUSSION TITLE
Have issues related to Typescript when upgraing to latest version of Vite and Vitest

### DISCUSSION THREAD

**Discussion by AlexMachin1997** [vitest-dev/vitest#3022] [Q&A]
Hi everyone,

After upgrading to the latest versions of Vite (^4.2.0) and Vitest (^0.29.3) I'm experiencing some Typescript issues related to the "test" property within the Vitest config.

I'm not entirely sure where I'm going, I've got the types included for both vite and vitest aswell as upgrading all the packages I'm aware of.

It would be great to get some insight into where I'm going wrong

Any comments or suggestions feel free to let me know in the thread below 😄 

Vite config: https://gist.github.com/AlexMachin1997/82bf906f5aecaa1c652735c3e1f9f04c

**Comment by sheremet-va** [comment]
Check your lockfile. You probably have several `vite` versions installed now
